docs/pages/api/access-plugin.mdx

```diff
@@ -237,37 +209,19 @@ The plugin imports the following third-party code:
 
 |Package|Description|
 |---|---|
-|`github.com/gravitational/teleport-plugins/lib`|Code shared between Teleport plugins, e.g., to run a background job for retrieving new events from the Teleport Auth Service.|
-|`github.com/gravitational/teleport-plugins/lib/watcherjob`|A library for writing programs that listen for audit events from the Auth Service.|
 |`github.com/gravitational/teleport/api/client`|A library for authenticating to the Auth Service's gRPC API and making requests.|
 |`github.com/gravitational/teleport/api/types`|Types used in the Auth Service API, e.g., Access Requests.|
 |`github.com/gravitational/trace`|Presenting errors with more useful detail than the standard library provides.|
 |`google.golang.org/api/option`|Settings for configuring Google API clients.|
 |`google.golang.org/api/sheets/v4`|The Google Sheets API client library, aliased as `sheets` in our program.|
 |`google.golang.org/grpc`|The gRPC client and server library.|
 
-### Make some initial declarations
+### Initial declarations
 
-After the `import` block, add the following declarations:
+First, we make some globally scoped declarations:
 
 ```go
-const (
-	proxyAddr     string = ""
-	initTimeout          = time.Duration(30) * time.Second
-	spreadSheetID string = ""
-)
-
-var requestStates = map[types.RequestState]string{
-	types.RequestState_APPROVED: "APPROVED",
-	types.RequestState_DENIED:   "DENIED",
-	types.RequestState_PENDING:  "PENDING",
-	types.RequestState_NONE:     "NONE",
-}
-
-type googleSheetsPlugin struct {
-	sheetsClient   *sheets.SpreadsheetsService
-	teleportClient *client.Client
-}
+(!examples/access-plugin-minimal/types.go!)
```

Comment:
nit: Is it possible to choose a specific line range to show so we can cut out the copyright comment at the top? Doesn't seem too awful to keep it, either.

Reply:
Sadly, no—the docs engine includes the entire partial